handlers/zipkin.go

```diff
@@ -45,11 +46,13 @@ func (z *Zipkin) ServeHTTP(rw http.ResponseWriter, r *http.Request, next http.Ha
 	existingTraceID := r.Header.Get(b3.TraceID)
 	existingSpanID := r.Header.Get(b3.SpanID)
 	if existingTraceID == "" || existingSpanID == "" {
-		traceID := idgenerator.NewRandom128().TraceID().String()
+		trace := idgenerator.NewRandom128().TraceID()
 
-		r.Header.Set(b3.TraceID, traceID)
-		r.Header.Set(b3.SpanID, traceID)
-		r.Header.Set(b3.Context, traceID+"-"+traceID)
+		span := fmt.Sprintf("%016x", trace.Low)
```

Comment:
Would `idgenerator.NewRandom128().SpanID(trace).String()` be preferable (or even reusing NewRandom128())?

Reply:
@stijnvet  - It appears @seankellner's suggestion is more appropriate that grabbing `trace.Low` for the span.  Will you consider updating your PR with this change?  Thanks!

Reply:
Yes, sorry it took so long to reply. Great suggestion